packages/neon-api/src/NetworkFacade.ts

```diff
@@ -45,7 +45,7 @@ export class NetworkFacade {
 
   async initialize(): Promise<void> {
     const response = await this.client.getVersion();
-    this.magicNumber = response.magic;
+    this.magicNumber = response.network;
```

Comment:
There are still a couple of places left in the code where there are `magicNumber` or `networkMagic`. On the C# side they renamed everything to `network`. You might want to think about if the remaining places are worth renaming or not.